src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,170 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> firstNode;
+    private Node<T> lastNode;
+
     @Override
     public void add(T value) {
+        Node<T> node;
+        if (size == 0) {
+            node = new Node<>(null, value, null);
+            firstNode = node;
+        } else {
+            node = new Node<>(null, value, null);
```

Comment:
can we initialize node once to avoid duplicates of code
`Node <T> node = new Node<> (null, value, null);`